### DISCUSSION TITLE
How can I make the scrollbar larger (not just during hover over)

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#13851] [Q&A] [answered]
I have a touch screen application that needs to display a scrollbar. Since it's a small screen (4.3") and touch only, the thin scrollbar line is barely visible:
![image](https://github.com/unoplatform/uno/assets/407941/e912c224-59b0-4556-be1c-ba33764d07d7)

I would like to make it larger, like what it looks like when I hover over (on my PC, Skia/GTK):
![image](https://github.com/unoplatform/uno/assets/407941/c13897be-7793-4daf-8366-9337a6a680e7)

But it has to be like this all the time, since there's no hover-over on the touch screen. How can I accomplish this? It's a vertical scrollbar in the `ScrollViewer` component. I've searched for a property to make the large view always visible but unable to find anything. Can I override the style? If so, how would I do that?

**Comment by wagenheimer** [comment]
Have you managed to increase it? I’ve been trying to expand it too, but I haven’t had any luck.

**Reply by HakanL** [reply, marked as answer]
It was a while ago, but I think this did the trick (definitely a hack):
```
    private void SettingsPage_Loaded(object sender, RoutedEventArgs e)
    {
#if HAS_UNO
        var sv = RepeaterScrollViewer;
        var vsb = (Control)sv.FindName("VerticalScrollBar");

        if (vsb != null)
        {
            var expandScrollBar = (object _, object _) =>
            {
                VisualStateManager.GoToState(vsb, "MouseIndicator", true);
                VisualStateManager.GoToState(vsb, "Expanded", true);
            };

            sv.PointerMoved += new Microsoft.UI.Xaml.Input.PointerEventHandler(expandScrollBar);
            sv.PointerEntered += new Microsoft.UI.Xaml.Input.PointerEventHandler(expandScrollBar);
            vsb.PointerExited += new Microsoft.UI.Xaml.Input.PointerEventHandler(expandScrollBar);
            vsb.PointerCaptureLost += new Microsoft.UI.Xaml.Input.PointerEventHandler(expandScrollBar);

            expandScrollBar(sender, e);
        }
#endif
    }
```
